Informed/Lesson Two/README.md

```diff
@@ -0,0 +1,381 @@
+<!-- Generics / Data Structures -->
+
+Welcome back! [Last time](../Lesson One), we took a look at Swift's syntax and Xcode playgrounds. We touched _very_ briefly on the idea of types. 
+
+Swift is a _strongly typed language_ and the type system is very robust. Combined with type inference, you can write expressive code without being repetitive. Many languages might define an integer variable as `int i = 0`, but in Swift, it's just `var i = 0`, and Swift figures out that `0` is an integer. 
+
+In this lesson, we're going to take a whirlwind tour through Swift's different types:
+
+- Structs
+- Classes
+- Enums
+- Tuples
+
+Finally, we'll take a deeper look at generics. Let's get started!
+
+----------------
+
+**Structs** in Swift are containers for other data, similar to structs in C. This is a valid Swift struct.
+
+```swift
+struct Artist {
+    let name: String
+}
+```
+
+What different from C, though, is Swift structs can also have functions on them. We'll see how shortly. 
+
+Once we have our `Artist` struct, how can we instantiate it? Well, Swift provides structs with default initializers – sometimes called constructors. To create an artist, you'd do the following. 
+
+```swift
+let artist = Artist(name: "Orta")
+```
+
+Very cool. If we added more fields to the `Artist` struct, we'd need to update our calls to the initializer. 
+
+Structs can have functions on them, too. Consider the following `Artwork` struct. 
+
+```swift
+struct Artwork {
+    let name: String
+    let artist: Artist
+
+    func getArtistName() -> String {
+        return artist.name
+    }
+}
+```
+
+It has a function that returns the artwork's artist's name. It seems a bit silly to have that as a function, though, since it's really a computed property. Swift let's us define those, too. 
```

Comment:
Yeah, gonna add a quick explanation near the beginning. Properties are pretty foundational.